tools/cldr-apps/src/org/unicode/cldr/web/MailSender.java

```diff
@@ -425,26 +421,43 @@ public void run() {
                     UserRegistry.User toUser = getUser(to);
                     if (toUser == null) {
                         String why;
-                        int badCount;
                         UserRegistry.User u = CookieSession.sm.reg.getInfo(to);
                         if (u != null && (UserRegistry.userIsLocked(u) || UserRegistry.userIsExactlyAnonymous(u))) {
                             why = "user " + u + " is locked or anonymous";
                         } else {
                             why = "user (#" + to + ") does not exist";
                         }
-                        rs.updateInt("try_count", (badCount = (rs.getInt("try_count") + 999))); // Don't retry.
+                        rs.updateInt("try_count", rs.getInt("try_count") + 999); // Don't retry.
                         rs.updateString("audit", why);
                         rs.updateTimestamp("try_date", sqlnow);
                         rs.updateRow();
                         conn.commit();
                         System.out.println("Abandoning mail id # " + lastIdProcessed + " because: " + why);
                         processMail();
                         return;
-                    } else if (to > 1) {
-                        ourMessage.addRecipient(MimeMessage.RecipientType.TO, new InternetAddress(toUser.email, toUser.name));
+                    }
+
+                    /*
+                     * The "from" address does not depend on the sender, due in part to problems with
+                     * authenticity checking schemes (such as DMARC). Instead it is a unicode.org
+                     * address, specific to the recipient user's organization, so that any bounce messages
+                     * get redirected (forwarded) back to the managers or tech-committee members for that
+                     * organization. Example: cldr-tc-reply-xyz@unicode.org, where xyz is replaced by
+                     * the organization name (lowercase, no spaces). CLDR_FROM in cldr.properties is obsolete.
+                     * References:
+                     *   https://unicode-org.atlassian.net/browse/CLDR-6334
+                     *   https://unicode-org.atlassian.net/browse/CLDR-10340
+                     */
+                    String fromEmail = "cldr-tc-reply-" + toUser.voterOrg() + "@unicode.org";
```

Comment:
I'd think these two pieces could be two config items, perhaps `CLDR_MAIL_FROM_PREFIX` and `CLDR_MAIL_FROM_SUFFIX`.

Could be a follow on PR.

Reply:
Do you see a reason we might want to reconfigure them?

Reply:
There might be multiple instances sometime. and in any event, we wouldn't want someone pulling down the code and generating mail to the address.  I think it's best left as configuration data rather than hard coded.

Reply:
Good point, at least for the suffix, or the domain name. We do have "CLDR_SMTP=unicode.org" currently on the production server, but CLDR_SMTP probably shouldn't be assumed always to match the desired domain for these email addresses.